package.json

```diff
@@ -63,7 +62,8 @@
         "test": "jest",
         "coverage": "yarn test --coverage",
         "analyse:unused-exports": "ts-node ./scripts/analyse_unused_exports.ts",
-        "analyse:webpack-bundles": "webpack-bundle-analyzer webpack-stats.json webapp"
+        "analyse:webpack-bundles": "webpack-bundle-analyzer webpack-stats.json webapp",
+        "update:jitsi": "curl -s https://meet.element.io/libs/external_api.min.js > ./res/jitsi_external_api.min.js"
```

Comment:
curl doesn't come preinstalled on Windows :D

Reply:
Right, this might be fine though because this isn't executed during builds anymore and we shouldn't actually have to run it locally at all with the workflow?